touchforms/backend/xformserver.py

```diff
@@ -165,7 +165,7 @@ def handle_request(content, server):
         session_id = content['session-id']
 
     action = content['action']
-    logger.info('Received action %s for session %s' % (action, session_id))
+    logger.info('event=received action=%s session=%s' % (action, session_id))
```

Comment:
what do you mean by removing the spaces from the events? i went half way to implement the [datadog canonical log format](http://docs.datadoghq.com/guides/logs/) but decided i'd probably need to make a new logger for that since it uses unix timestamps for dates which isn't very useful when you're looking at logs. i figured this would be easy enough to parse `split()` and then split on the `=` sign